azure-toolkit-libs/azure-toolkit-springcloud-lib/src/main/java/com/microsoft/azure/toolkit/lib/springcloud/SpringCloudApp.java

```diff
@@ -149,4 +151,23 @@ public SpringCloudPersistentDisk getPersistentDisk() {
     public boolean isPersistentDiskEnabled() {
         return Objects.nonNull(this.getPersistentDisk());
     }
+
+    public void enableRemoteDebugging(int port) {
+        DeploymentsClient client = Utils.getDeploymentsClient(this.getSubscriptionId());
+        RemoteDebuggingPayload payload = new RemoteDebuggingPayload().withPort(port);
+        client.enableRemoteDebugging(this.getResourceGroupName(), Objects.requireNonNull(this.getParent().getRemote()).name(),
+                this.getName(), this.getActiveDeploymentName(), payload);
+    }
+
+    public void disableRemoteDebugging() {
+        DeploymentsClient client = Utils.getDeploymentsClient(this.getSubscriptionId());
+        client.disableRemoteDebugging(this.getResourceGroupName(), Objects.requireNonNull(this.getParent().getRemote()).name(),
+                this.getName(), this.getActiveDeploymentName());
+    }
+
+    public boolean isRemoteDebuggingEnabled() {
+        DeploymentsClient client = Utils.getDeploymentsClient(this.getSubscriptionId());
+        return client.getRemoteDebuggingConfig(this.getResourceGroupName(), Objects.requireNonNull(this.getParent().getRemote()).name(),
+                this.getName(), this.getActiveDeploymentName()).enabled();
+    }
```

Comment:
these methods should be moved to `SpringAppDeployment`, and it already have a method to get the client. there is no need to make a new `Utils.getDeploymentClient`.